src/w2toolbar.js

```diff
@@ -143,7 +143,7 @@
                 $(this.box).find('#tb_'+ this.name +'_item_'+ w2utils.escapeId(it.id)).remove();
                 // remove from array
                 var ind = this.get(it.id, true);
-                if (ind != null) this.items.splice(ind, 1);
+                if (ind >= 0) this.items.splice(ind, 1);
```

Comment:
I think it makes no difference. Condition
```js
if (ind != null) 
```
(note I did not use if (ind !== null) ) will always be true if ind is null or undefined. If ind is 0, the condition will not be true. But for the get function, if it does not find the item, it will return null.